lib/components/form/connect-location-field.js

```diff
@@ -23,9 +23,15 @@ export default function connectLocationField (StyledLocationField, options = {})
     const activeSearch = getActiveSearch(state)
     const query = activeSearch ? activeSearch.query : currentQuery
 
+    const geocoderConfig = config.geocoder
+    if (state.otp.location.currentPosition.coords) {
+      const {latitude: lat, longitude: lon} = state.otp.location.currentPosition.coords
```

Comment:
Reuse variable `currentPosition` already declared on L22.

Reply:
Can't believe I missed that! Thanks for catching and updated in 50deca7318c6444da0ded3e0b97a3d5c23dd8aec